packages/graphiql/package.json

```diff
@@ -13,21 +13,26 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
+  "module": "esm/index.js",
+  "types": "index.d.ts",
   "files": [
     "dist",
-    "graphiql.js",
```

Comment:
Are we still bundling non-minified GraphiQL or not? (We should - for better error messages at least.)

Reply:
we are yeah, ill be fixing this